openBCIWifi.js

```diff
@@ -4,17 +4,19 @@ const _ = require('lodash');
 const util = require('util');
 // Local imports
 const OpenBCIUtilities = require('@openbci/utilities');
-const obciUtils = OpenBCIUtilities.utilities;
-const k = OpenBCIUtilities.constants;
-const obciDebug = OpenBCIUtilities.debug;
+const obciUtils = require('@openbci/utilities/dist/utilities');
+const k = require('@openbci/utilities/dist/constants');
+const obciDebug = OpenBCIUtilities.Debug;
```

Comment:
what the reason for changing these imports? did they not work the way they were before?

code looks good otherwise, and it works for me!

Reply:
both work, it doesn't need to be changed